examples/cat.js

```diff
@@ -0,0 +1,23 @@
+let StringHelper = require('../index')
+
+let helper = new StringHelper()
+let result = helper.cat('this is the first line', '\n')
+    .cat('here is the second\n')
+    .cat(['and then', ' the third'], '\n')
+    .cat('now').cat(function () {
```

Comment:
Alright, used a shorter syntax, but added a note in the documentation about the usage of arrow functions and access to the helper. 5ea3ee1